content/models.py

```diff
@@ -135,3 +135,31 @@ class FooterInfo(models.Model):
 
 	def __str__(self):
 		return self.title
+
+class DonateInfo(models.Model):
```

Comment:
why do we have both `DonateInfo` and `DonateSection`

Seems like you can merge them into one? Or am I missing something